snowpenguin/django/recaptcha2/templates/snowpenguin/recaptcha/recaptcha_init.html

```diff
@@ -1 +1,33 @@
-<script src="https://www.google.com/recaptcha/api.js{% if explicit %}?onload=djangoRecaptchaOnLoadCallback&render=explicit{% if language %}&hl={{ language }}{% endif %}{% elif language %}?hl={{ language }}{% endif %}" async defer></script>
\ No newline at end of file
+<script src="https://www.google.com/recaptcha/api.js{% if explicit %}?render=explicit{% if language %}&hl={{ language }}{% endif %}{% elif language %}?hl={{ language }}{% endif %}" async defer></script>
+{% if explicit %}
+<script>
+    window.addEventListener('load', function() {
+      var captchas = [].slice.call(document.querySelectorAll('.g-recaptcha'));
+
+      captchas.map(function(el) {
+        var containerID = el.id;
+        var options = {'sitekey': el.dataset.sitekey};
+
+        if (el.dataset.theme) {
```

Comment:
yeah loop it the obvious missing code here 🤦‍♂️ 
but no arrow function because of browsers incompatibility and `el.dataset`